### PR TITLE
Manage your CWS item

### PR DESCRIPTION
**Pull request by AmySteam**
This page is a draft

**Staged preview**
[Manage your CWS item](https://deploy-preview-1224--developer-chrome-com.netlify.app/docs/webstore/manage/)

### CODE REVIEW

File: site/en/docs/webstore/manage/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+To stay informed of your item's status, you can:
+
+-  **Enable email notifications**. There are mandatory emails like take down or rejection notifications that are enabled by default. To receive notification for published or staged items, you can enable email notifications in the Account page. 
+ 
+    {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/FgPIwRl3QEbNowNV1zRn.png", alt="How to enable email notifications", width="658", height="219" %}
+
+-  **Check your publisher email often**. After enabling email notifications check your inbox and spam folder, just in case.
+   
+## Comply with Chrome Web Store policies
+To ensure a positive experience for your users follow the [developer program policies][dev-policies]. These policies may change, so keep up with [new announcements][whats-new].
```
Comment (samthor) [inline site/en/docs/webstore/manage/index.md:34]: nit: insert blank line above

File: site/en/docs/webstore/manage/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+All extensions go through an automated review process. In some instances a manual review is required, specially when sensitive permissions are requested. For this reason review times and/or approval times can take longer. A Chrome Web Store item goes through several stages, as depicted in the following diagram:
+
+{% Aside %}
+Insert diagram here
+{% endAside %}
+
+## Check your item's status
+ 
+ The status of your item appears in the [developer dashboard][dev-dashboard] next to each item. The status can be Published, Pending or Rejected.
```
Comment (samthor) [inline site/en/docs/webstore/manage/index.md:21]: nit: remove leading whitespace

File: site/en/docs/webstore/manage/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+date: 2021-08-17
+description: >
+  How to manage your extension in the Chrome Web Store.
+---
+
+This page describes how to manage an extension that was previously submitted on the Chrome Web Store.
+
+## About the lifecycle of an item in the Chrome Web Store 
+
+All extensions go through an automated review process. In some instances a manual review is required, specially when sensitive permissions are requested. For this reason review times and/or approval times can take longer. A Chrome Web Store item goes through several stages, as depicted in the following diagram:
```
Comment (samthor) [inline site/en/docs/webstore/manage/index.md:13]: This is totally bikeshedded, but could you put each new sentence on its own line? I don't want wrapping, but whole paragraphs makes restructuring still a bit tricky.

File: site/en/docs/webstore/manage/index.md [hunk 4]
```diff
@@ -3,22 +3,22 @@ layout: "layouts/doc-post.njk"
...
+  How to manage an extension or theme ("item") in the Chrome Web Store.
 ---
 
-This page describes how to manage an extension that was previously submitted on the Chrome Web Store.
+This page describes how to manage an extension or theme ("item") that was previously submitted on the Chrome Web Store.
 
 ## About the lifecycle of an item in the Chrome Web Store 
 
-All extensions go through an automated review process. In some instances a manual review is required, specially when sensitive permissions are requested. For this reason review times and/or approval times can take longer. A Chrome Web Store item goes through several stages, as depicted in the following diagram:
+All Chrome Web Store items go through an automated review process. In some instances a manual review is required, specially when sensitive permissions are requested.
```
Comment (jpmedley) [inline site/en/docs/webstore/manage/index.md]:
```suggestion
All Chrome Web Store items go through an automated review process. In some instances a manual review is required, especially when sensitive permissions are requested. 
```

File: site/en/docs/webstore/manage/index.md [hunk 5]
```diff
@@ -3,22 +3,22 @@ layout: "layouts/doc-post.njk"
...
 
-{% Aside %}
-Insert diagram here
-{% endAside %}
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/yakIoAzWEchO8urosQLs.png", alt="Diagram of lifecycle of a Chrome Web Store item", width="800", height="177" %}
 
 ## Check your item's status
  
- The status of your item appears in the [developer dashboard][dev-dashboard] next to each item. The status can be Published, Pending or Rejected.
+The status of your item appears in the [developer dashboard][dev-dashboard] next to each item. The status can be Published, Pending or Rejected.
```
Comment (jpmedley) [inline site/en/docs/webstore/manage/index.md]:
```suggestion
The status of your item appears in the [developer dashboard][dev-dashboard] next to each item. The status can be Published, Pending, or Rejected.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Really clean. I just found a few typos.

**Comment by AmySteam** [comment]
@jpmedley, applied your changes! Made a minor tweak, can you PTAL? Thanks! [Staged Preview](https://deploy-preview-1224--developer-chrome-com.netlify.app/docs/webstore/manage/)

**Review by jpmedley — APPROVED** [review]

**Comment by jpmedley** [comment]
Within the Chrome DevRel team, authors merge their own PRs. Have you been doing that or do you need me to.

**Comment by AmySteam** [comment]
@jpmedley I have been merging! Thanks 👍

**Pull request merged**